lib/spack/docs/getting_started.rst

```diff
@@ -1322,3 +1322,69 @@ To ensure that Spack does not autodetect the Cray programming
 environment, unset the environment variable ``MODULEPATH``. This
 will cause Spack to treat a linux container on a Cray system as a base
 linux distro.
+
+.. _windows_support:
+
+----------------
+Spack On Windows
+----------------
+
+You will require a MSVC compiler, Git and CMake in order to use Spack on Windows.
```

Comment:
Are git and cmake necessary when using the installer?

Reply:
Yes, for right now, you'll need git to pull feature/windows and cmake, I believe isn't buildable from Spack without another cmake to build it. Hopefully this list of requirements gets smaller in later versions.

Reply:
Ah, okay.